sdcm/tester.py

```diff
@@ -3050,6 +3050,11 @@ def tearDown(self):
         self.stop_resources()
         self.get_test_failures()
 
+        with silence(parent=self, name='closing decoding queue as needed'):
+            self.test_config.BACKTRACE_DECODING = self.params.get('backtrace_decoding')
```

Comment:
this line looks redundant

Reply:
right, I wasn't reading it carefully enough...